src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/audit/AuditInterceptor.kt

```diff
@@ -18,6 +21,11 @@ class AuditInterceptor(private val storedProcedureRepository: StoredProcedureRep
       return proceedingJoinPoint.proceed()
     } finally {
       storedProcedureRepository.clearAudit()
+      log.debug("annotatedEndpoint() : Cleared audit. Transaction is ${if (TransactionSynchronizationManager.isActualTransactionActive()) "" else "not "}active")
```

Comment:
So now if clearAudit() throws an exception, the transaction manager will catch it and rollback, preventing any db change